packages/jest-jasmine-adapter/src/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const {addMatchers} = require('jest-matchers');
+
+// A single place to store everything jasmine related. (hacks, patches,
+// backward compatibility functions, etc.)
+const setup = () => {
+  const jasmine = global.jasmine || require('jasmine');
```

Comment:
we shouldn't do `require('jasmine')` here, it'll just break because there is no `jasmine` module in the vm sandbox. We can also always expect that the global.jasmine will be there, so this stuff seems unnecessary.